scripts/pipelines/gcloud/templates/deploy/deploy-pipeline.cfg

```
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Mandatory flags.
+mandatoryFlags="$pipelineName,$localDirectory,$packagePipelineName,$envProvisionPipelineName,$k8sProvider,$k8sNamespace,$k8sDeployFiles,"
+# Path to the templates.
+templatesPath="scripts/pipelines/gcloud/templates/deploy"
+# Path to the common templates folder
+commonTemplatesPipelinePath="scripts/pipelines/common/templates/deploy"
+# YAML file name.
+yamlFile="deploy-pipeline.yml"
+# Script name.
+scriptFile="deploy.sh"
+# Source branch.
+sourceBranch="feature/deploy-pipeline"
+# Description of the trigger
+triggerDescription="Trigger for deploy pipeline"
+# Branch trigger
+branchTrigger='.*'
+# Setup Environment script
+setupScriptFile="deploy-setup-environment.sh"
+# Previous pipeline
+previousPipelineyaml="*package-pipeline.yml"
```

```suggestion
previousPipelineyaml="package-pipeline.yml"
```
Now there is only one package-pipeline for every case, wildcard no longer needed.